device-types/APC/SMT1000RMI2UC.yaml

```diff
@@ -0,0 +1,28 @@
+---
+manufacturer: APC
+model: Smart-UPS 1000VA, Rack Mount, LCD 230V with SmartConnect Port
+slug: smart-ups-1000va-rack-mount-lcd-230-with-smartconnect-port
```

Comment:
Please append `apc-` to the slug. I suggest you change this to `apc-smt1000rmi2uc`